Cython/Setuptools.py

```diff
@@ -0,0 +1,36 @@
+from Cython.Build import cythonize
+from setuptools import Command
+
+
+def cython_modules(dist, attr, value):
+    assert attr == 'cython_modules'
+
+    if not dist.ext_modules:
+        dist.ext_modules = []
+
+    dist.ext_modules += cythonize(value, replace_extension=True)
+
+
+def cython_manual_modules(dist, attr, value):
+    pass
+
+
+class BuildPyx(Command):
```

Comment:
Should we be subclassing `build_ext`?

Reply:
I think not, because this command should run *before* `build_ext`.